templates/execute-query-send-result.json

```diff
@@ -0,0 +1,305 @@
+{
```

Comment:
I think you also need to update the `manifest.json` file to include this template.